frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -33,6 +37,9 @@ export const TextRow = ({
   const [textIdValue, setTextIdValue] = useState(textId);
   const [textEntryValue, setTextEntryValue] = useState(textData?.value || '');
   const [keyError, setKeyError] = useState('');
+  const [isConfirmDeleteOpen, setIsConfirmDeleteOpen] = useState(false);
```

Comment:
This part doesn't need to be set. The popover-component handles its open state inside the component.

Reply:
But we do need that state to re-render the `<Popover/>` to actually open when the user clicks on the "Delete" button. Do you have another way to achieve this without this state, @mijohansen? :)

![image](https://user-images.githubusercontent.com/46874830/222417850-b9c8b26e-7692-4360-9b35-9d736b4bf550.png)